### Problema 1

Calcular la actividad de $1 \text{g}$ de $^{266}Ra$.

### Solución



El número de núcleos de $^{206}\text{Ra}$ en $1\text{g}$, se puede hallar utilizando la definición de masa molar y el número de Avogadro. La masa atómica del $^{206}\text{Ra}$ es de $266\text{u}$, es decir que por cada $1 \text{mol}$ de $^{266}\text{Ra}$ hay $266\text{g}$. El número de avogradro nos dice que hay $6.022 \times 10^{23}$ átomos por cada mol, por lo que el número de núcleos 

$$N_{266\text{Ra}} = \frac{1g}{266\frac{g}{mol}} \frac{6.022\times^{23}}{\text{mol}} = 2.64 \times 10^{21} = 2.263 \times 10 ^{21}$$

Por otra parte la actividad esta definida por:

$$ \frac{dN}{dt} = -\lambda N \qquad \lambda = \frac{\log(2)}{\tau}$$

Con $\tau$ la vida media de una especie. La vida media de este isotopo del Radón es de $1600$ años, o aproximadamente $5.04 \times 10^{10}\text{s}$. Reemplazando, la actividad del Radón será:

$$ -\frac{dN}{dt} = 3.858 \times 10^{10} \text{s} \equiv 1 \text{Ci}$$

En donde $1 \text{Ci}$ es la unidad fundamental para las tasas de desintegración y es llamada Curie.


### Problema 2

La abundancia actual de los isótopos $^{235}\text{U}$ y $^{238}\text{U}$ es de $0,72\%$ y $99,27\%$ respectivamente. Estimar la edad de la tierra haciendo la suposición de que en el génesis la población relativa de ambos isótopos era la misma. 

Las vidas medias de los isotopos de $^{235}\text{U}$ y $^{238}\text{U}$ son respectivamente:
- $\tau^{^{235}\text U} = 7.04 \times 10^8 \text{años}$
- $\tau^{^{238}\text U} = 4.468 \times 10^9 \text{años}$

### Solución

Resolviendo la ecuación de decaimiento obtenemos que:

$$ \frac{dN}{dt} = -\lambda N \Rightarrow \int_{N_i}^{N_f} \frac{dN}{N} = \log(\frac{N_f}{N_i}) = - \lambda \int_{t_i}^{t_f} dt$$

$$ \Rightarrow \log(\frac{N_i}{N_f}) = \lambda \Delta t \Rightarrow N_i = N_f e^{\lambda \Delta t}$$

En donde $\Delta t$ es el tiempo en la tierra y $\lambda = \frac{\log(2)}{\tau}$ la vida media de un isotopo. Si bien no sabemos el valor de $N_i$, para los isótopos propuestos, si sabemos que este valor era igual para ambos y que el tiempo transcurrido fue el mismo. Luego:

$$ N_f^{^{238}\text{U}} e^{\lambda^{^{238}\text{U}} \Delta t} = N_f^{^{235}\text{U}} e^{\lambda^{^{235}\text{U}} \Delta t}$$

$$ \Rightarrow 0.9927 e^{\lambda^{^{238}\text{U}} \Delta t} = 0.0072 e^{\lambda^{^{235}\text{U}} \Delta t} \Rightarrow \frac{0.9927}{0.0072} = e^{(\lambda^{^{238}\text{U}} - \lambda^{^{235}\text{U}})\Delta t} $$

$$ \Rightarrow \Delta t = \frac{\log(\frac{0.9927}{0.0072})}{(\lambda^{^{238}\text{U}} - \lambda^{^{235}\text{U}})} = \frac{\log(\frac{0.9927}{0.0072})}{\log(2)} \left(\frac{1}{\tau^{^{238}\text{U}}} - \frac{1}{\tau^{^{235}\text{U}}} \right)^{-1}$$

Reemplazando:

- $\tau^{^{235}\text U} = 7.04 \times 10^8 \text{años}$
- $\tau^{^{238}\text U} = 4.468 \times 10^9 \text{años}$

In [6]:
from math import log

dt = lambda tau1, tau2: log(0.9927/0.0073)/(1/tau1-1/tau2)

tau235U = 7.04e8
tau238U = 4.468e9

print(dt(tau235U, tau238U)/10**9, 'x10^9 años')

4.1052873391138025 x10^9 años


Obteniéndose que la edad de la tierra es de aproximadamente $4.1 \times 10^9 \text{años}$. 

### Problema 3

Considerar que un nucleido inestable $X_1$ decae en el nucleido $X_2$ con una constante de decaimiento $\lambda_1$. A su vez, el nucleido $X_2$, tambien inestable, decae en el nucleido $X_3$ con una constante de decaimiento $\lambda_2$. El nucleido $X_3$ es estable. Sabiendo que inicialmente había $N_0$ nucleidos del tipo $X_1$, encontrar la cantidad de cada uno de estos en función del tiempo.

### Solución

En el enunciado se describen una cascada de reacciones nucleares:


$$\frac{dN_1}{dt} = -\lambda_{1} N_1$$
$$\frac{dN_2}{dt} = \lambda_{1} N_1 - \lambda_{2}N_2$$
$$\frac{dN_3}{dt} = \lambda_{2} N_2$$

En donde el signo negativo implica que la cantidad de nucleidos disminuye con el tiempo y el signo positivo que aumenta. Se tienen las siguientes restricciones:

$$N_1 + N_2 + N_3 = N_0$$

$$\Rightarrow \frac{dN_1}{dt} + \frac{dN_2}{dt} + \frac{dN_3}{dt} = 0$$

Con condición inicial $N_1(0) = N_0$. Notemos que la solucíon a la primera ecuación es:

$$ N_1(t) = N_0 e^{-\lambda_1 t}$$

Reemplazando se obtiene la ecuación diferencial:

$$ \frac{dN_2}{dt} = \lambda_{1}  N_0 e^{-\lambda_1 t} - \lambda_{2}N_2$$

Se propone una solución homogénea:

$$ \frac{dN^{h}_2}{dt} = - \lambda_{2}N^{h}_2 \Rightarrow N_2^h = C e^{-\lambda_{2} t}$$

Y una solución particular:

$$ N_2^p = A(t) e^{-\lambda_2 t}$$

$$ \frac{dN_2^p}{dt} = A'(t) e^{-\lambda_2 t} - \lambda_2 A(t) e^{-\lambda_2 t} = \lambda_1 N_0 e^{-\lambda_1 t} - \lambda_2 N_2^p(t)$$

$$ \Rightarrow A'(t) e^{-\lambda_2 t} = \lambda_1 N_0 e^{-\lambda_1 t} \Rightarrow A'(t) = \lambda_1 N_0 e^{(\lambda_2 - \lambda_1)t}$$ 

$$ \Rightarrow A(t) = \frac{\lambda_1 N_0}{\lambda_2 - \lambda_1} e^{(\lambda_2 - \lambda_1)t} + C$$

De modo que la solución general es:

$$ N_2(t) = C e^{-\lambda_{2} t} + \frac{\lambda_1 N_0}{\lambda_2 - \lambda_1} e^{-\lambda_1t} $$

Lluego $N_2(0) = 0$ lo que implica que:


$$C = - \frac{\lambda_1 N_0}{\lambda_2 - \lambda_1}$$

$$ \Rightarrow N_2(t) = \frac{\lambda_1 N_0 }{\lambda_2 - \lambda_1}(e^{-\lambda_1t} - e^{-\lambda_2t})$$


Hallamos ahora la solución para $N_3$:

$$ \frac{dN_3}{dt} = \lambda_{2} \frac{\lambda_1 N_0 }{\lambda_2 - \lambda_1}(e^{-\lambda_1t} - e^{-\lambda_2t})$$

Integrando:

$$ \int dN_3 = \frac{\lambda_2 \lambda_1 N_0}{\lambda_2 - \lambda_1}\int (e^{-\lambda_1t} - e^{-\lambda_2t})dt$$

$$ N_3(t) = \frac{\lambda_2 \lambda_1 N_0}{\lambda_2 - \lambda_1} (\frac{e^{-\lambda_2t}}{\lambda_2} - \frac{e^{-\lambda_1t}}{\lambda_1} ) + C$$

$$ N_3(0) = 0 \Rightarrow \frac{\lambda_2 \lambda_1 N_0}{\lambda_2 - \lambda_1} (\frac{1}{\lambda_2} - \frac{1}{\lambda_1} ) + C= -N_0 + C = 0$$

$$ \Rightarrow N_3(t) = N_0 \left(1 - \frac{\lambda_2 e^{-\lambda_1t} - \lambda_1 e^{-\lambda_2t}}{\lambda_2 - \lambda_1} \right) $$

### Problema 5

Un método de producir un nucleido radiactivo consiste en colocar una muestra de una determinada sustancia en el interior de un reactor nuclear. Los nucleidos radiactivos se producen como consecuencia de la captura de un neutrón por los núcleos de la sustancia.

En un reactor nuclear se coloca $^{59}\text{Co}$. El nuevo nucleido se produce a razón de $g$ núcleos por segundo. Calcular el número de núcleos radiactivos producido en función del tiempo.


### Solución
La reacción nuclear del cobalto al capturar un neutrón es:

$$^{59}_{27}\text{Co} + n \rightarrow ^{60}_{27}\text{Co}$$

Esta reacción implican los siguientes cambios en las poblaciones:

$$\frac{dN_{59}}{dt} = -g - \lambda N_{59} \qquad \frac{dN_{60}}{dt} = g - \lambda N_{60} $$

Buscamos resolver la siguiente ecuación diferencial. Proponemos la solución homogénea $N_{60}^h$ tal que:

$$\frac{dN_{60}^h}{dt} = -\lambda N_{60}^h \Rightarrow N_{60}^h = C e^{-\lambda t}$$

Buscamos ahora una solución particular $N_{60}^p$ tal que $N_{60}^p = A(t) e^{-\lambda t}$, con $A(t)$ una función a determinar. Reemplazando en la ecuación diferencial:

$$\frac{dN_{60}^p}{dt} = A'(t) e^{-\lambda t} - \lambda A(t) e^{-\lambda t} = g - \lambda N_{60}^p$$

$$\Rightarrow A'(t) = g e^{\lambda t} \Rightarrow A(t) = g \frac{e^{\lambda t}}{\lambda} + C$$

Luego la solución general es:

$$N_{60}(t) = C e^{-\lambda t} + \frac{g}{\lambda}$$

Dado que al inicio de la reacción, no hay nucleos de $^{60}\text{Co}$, se tiene que $N_{60}(0) = 0$ lo que implica que $C = -\frac{g}{\lambda}$. El numero de núcleos de $^{60}\text{Co}$ en función del tiempo es:

$$N_{60}(t) = \frac{g}{\lambda} (1 - e^{-\lambda t})$$

### Problema 6

La razón másica entre el radioisótopo $^{14}\text{C}$ y el estable $^{12}\text{C}$ es de $1.5 \times 10 ^{-12}$. Encontrar el número de desintegraciones por segundo en $1\text{g}$ de materia viva.

En una muestra de $64\text{g}$ de carbón se observan 2 desintegraciones por segundo. Calcular la edad de la muestra.

Considerar que la vida media del $^{14}\text{C}$ es de $5700$ años.


### Solución

El $^{12}\text{C}$ es estable, y por lo tanto no se consideran desintegraciones. El $^{14}\text{C}$ decae en $^{14}\text{N}$ a través de un proceso de desintegración beta, es decir se produce la reacción:

$$^{14}_{6}\text{C} \rightarrow ^{14}_{7}\text{N} + \beta^-$$

Que la razón másica entre el $^{14}\text{C}$ y el $^{12}\text{C}$ sea de $1.5 \times 10^{-12}$ implica que:

$$\frac{m^{^{14}\text{C}}}{m^{^{12}\text{C}} + m^{^{14}\text{C}}} = 1.5 \times 10^{-12}$$

Para una muestra de $1\text{g}$ de materia viva, la cantidad de $^{14}\text{C}$ es de $1.5 \times 10^{-12} \text{g}$. Si la masa molar del $^{14}\text{C}$ es de $14\frac{\text{g}}{\text{mol}}$, entonces la cantidad de núcleos de $^{14}\text{C}$ es de:

$$N^{^{14}\text{C}} = \frac{1.5 \times 10^{-12}}{14} \times 6.022 \times 10^{23} = 64.52 \times 10^{9}$$

Calculamos ahora la actividad de la muestra. La actividad esta dada por:

$$\frac{dN}{dt} = -\lambda N = -\frac{\log(2)}{\tau} N$$

In [ ]:
from math import log

def years_to_seconds(time):
    return time * 365.2525 * 24 * 3600

def hz_to_curie(frequency):
    return frequency / 3.7e10


N14C = 64.52e9
tau14C = 5730 # years
activity = lambda N, tau: log(2)*N/tau
print(activity(N14C, years_to_seconds(tau14C)), 'desintegraciones por segundo')
print(hz_to_curie(activity(N14C, years_to_seconds(tau14C))), 'Ci')

0.24731944209079143 desintegraciones por segundo
6.684309245697066e-12 Ci


Se tiene que la actividad de la muestra sera de $0.24$ desintegraciones por segundo. El hecho de que la muestra sea de materia viva, implica que la cantidad de $^{14}\text{C}$ es igual a la de la atmosfera Dado que el proceso de fotosíntesis incorpora el átomo radiactivo en los organismos autótrofos, y estos son consumidos luego por los heterótrofos.

Se tiene ahora que la actividad de una muestra de $64\text{g}$ de carbono es de $2$ desintegraciones por segundo, es decir $2/64 = 0.03125$ desintegraciones por segundo por gramo de la muestra. Una actividad menor a la obtenida en el resultado anterior, es decir al transcurrir el tiempo una fracción de los núcleos de $^{14}\text{C}$ se desintegraron y la actividad disminuyo con respecto a la de la materia viva.

Reemplazando estos valores en la ecuación de actividad, podemos obtener el número de átomos de $^{14}\text{C}$ en $1\text{g}$ de la muestra.

$$0.03125 = \frac{\log(2)}{\tau} N^{^{14}\text{C}} \Rightarrow  N^{^{14}\text{C}} = \frac{0.03125 \tau}{\log(2)}$$


Por otra parte si se resuelve la ecuación diferencial de desintegración, se obtiene que:

$$ \int_{N_0}^{N} \frac{dN}{N} = -\lambda \int_{0}^{t} dt \Rightarrow \log(\frac{N}{N_0}) = -\lambda t $$

$$ \Rightarrow t = \frac{\tau}{\log(2)} \log{\frac{N_0}{N}}$$

Luego si se toma el numero de núcleos de $^{14}\text{C}$ en la muestra de materia viva como $N_0$, podemos obtener el tiempo transcurrido desde que la materia viva desintegro atomos de $^{14}\text{C}$ hasta el momento de la prueba. Esto es:

In [30]:
from math import log

def years_to_seconds(time):
    return time * 365.2525 * 24 * 3600

def seconds_to_years(time):
    return time / 365.2525 / 24 / 3600

tau14C = 5730 # years
N14C = 64.52e9 #N0
Nm14C = 2/64*years_to_seconds(tau14C)/log(2) #N
t = lambda tau, N0, N: tau*log(N0/N)/log(2)

print(seconds_to_years(t(years_to_seconds(tau14C), N14C, Nm14C)), 'años')


17100.884526597005 años


Por lo que la muestra tendra 17101 años aproximadamente